frontend/site-registry/src/app/features/details/parcelDescriptions/getParcelDescriptions.ts

```diff
@@ -0,0 +1,96 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+import { GRAPHQL } from '../../../helpers/endpoints';
+import { getAxiosInstance } from '../../../helpers/utility';
+import {
+  IParcelDescriptionDto,
+  IParcelDescriptionResponseDto,
+} from './parcelDescriptionDto';
+import { format } from 'date-fns';
+
+export async function getParcelDescriptions(
```

Comment:
This should be part of redux & data should be managed in redux . getParcelDescriptions should be implemented using createAsyncThunk